modules/util/create.py

```diff
@@ -809,61 +954,64 @@ def create_optimizer(
             )
 
     if state_dict is not None and optimizer is not None:
-        if 'param_group_mapping' not in state_dict:
+        if "param_group_mapping" not in state_dict:
             # Old method of loading the optimizer state. This only works if the param groups did not change.
             for i, params in enumerate(parameters):
-                state_dict['param_groups'][i]['lr'] = params['lr']
-                state_dict['param_groups'][i]['initial_lr'] = params['initial_lr']
+                state_dict["param_groups"][i]["lr"] = params["lr"]
+                state_dict["param_groups"][i]["initial_lr"] = params["initial_lr"]
         else:
             # New method of loading the optimizer state. Each group is mapped by a unique name.
-            old_state = state_dict['state']
-            old_param_groups = state_dict['param_groups']
-            old_group_mapping = state_dict['param_group_mapping']
-            old_group_optimizer_mapping = state_dict['param_group_optimizer_mapping']
+            old_state = state_dict["state"]
+            old_param_groups = state_dict["param_groups"]
+            old_group_mapping = state_dict["param_group_mapping"]
+            old_group_optimizer_mapping = state_dict["param_group_optimizer_mapping"]
 
-            new_param_groups = optimizer.state_dict()['param_groups']
+            new_param_groups = optimizer.state_dict()["param_groups"]
             new_group_mapping = parameter_group_collection.unique_name_mapping
 
             state = {}
             param_groups = []
             state_index = 0
 
             for new_group_index, unique_group_name in enumerate(new_group_mapping):
-                if (unique_group_name in old_group_mapping and str(config.optimizer.optimizer) ==
-                        old_group_optimizer_mapping[old_group_mapping.index(unique_group_name)]):
+                if (
+                    unique_group_name in old_group_mapping
+                    and str(config.optimizer.optimizer)
+                    == old_group_optimizer_mapping[old_group_mapping.index(unique_group_name)]
```

Comment:
Similar thing here. shouldn't the line starting with `==` be indented more?

Reply:
Disabled formatter.